Source/JavaScriptCore/runtime/OptionsList.h

```diff
@@ -578,7 +578,7 @@ bool hasCapacityToUseLargeGigacage();
     v(Bool, useWebAssemblyIPIntEpilogueOSR, true, Normal, "Allow IPInt to tier up during function epilogues"_s) \
     v(Bool, webAssemblyIPIntTiersUpToBBQ, true, Normal, "Allow IPInt to tier up to BBQ"_s) \
     v(Bool, webAssemblyIPIntTiersUpToOMG, true, Normal, "Allow IPInt to tier up to OMG"_s) \
-    v(Bool, useInterpretedJSEntryWrappers, false, Normal, "Allow some JS->wasm wrappers to be replaced by jit-less versions."_s) \
+    v(Bool, useInterpretedJSEntryWrappers, true, Normal, "Allow some JS->wasm wrappers to be replaced by jit-less versions."_s) \
```

Comment:
Again for my own understanding: why just some?

Reply:
I forgot to update the documentation! Should be all now!